admin/cli/opts/opts.go

```diff
@@ -0,0 +1,8 @@
+package opts
```

Comment:
This may be completely wrong, so I'm looking for feedback here.

The reason is some commands may require a global configuration change such as not setting up clients for the status command https://github.com/percona/pmm/pull/934/files#diff-2648659f55861d624fd73ae93f0db417bf1de9af5f3370827952529639d657c7R89

I considered using it for passing in context as well but found a different approach.

Without a separate module I was running into cyclic dependency issue. I had to create a new module which does not require any `commands/*` to avoid the issue.

Happy to reconsider if there's a more elegant solution though.

Reply:
Originally I wanted to put everything under `admin/commands/cli.go`. This has the same issue with cyclic dependencies though because you'd be importing `admin/commands/management` which in turn imports `admin/commands`.

I could not put everything under `admin/cli` because then you would need to duplicate structures to configure the `Run()` methods for commands. 

My goal was to do as little refactoring to the existing commands as possible so it's easy to review this PR.
I don't like the directory structure either :) If there's a better way, I'm happy to refactor this.